packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue

```diff
@@ -24,7 +24,7 @@
         <code class="event-type-tag" v-for="tag in eventType.tags" :key="tag">{{ tag }}</code>
       </div>
       <div class="event-type-release">
-        Since: <a href="/docs/release-notes/">{{ eventType.info.release }}</a>
+        Since: <SmartLink :item="{link: '/docs/release-notes/', text: eventType.info.release}" />
```

Comment:
Is it possible for `SmartLink` to pull the children of the `SmartLink` component instead of having to pass `item.text`? e.g.
```js
<SmartLink :item="{link: '/docs/release-notes/'}">{{eventType.info.release}}</SmartLink>
```

Reply:
Good idea... Considering I already handled the `slot` for other cases, this will be cleaner.